src/go/k8s/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -104,6 +104,8 @@ type ClusterSpec struct {
 	// Replicas determine how big the cluster will be.
 	// +kubebuilder:validation:Minimum=0
 	Replicas *int32 `json:"replicas,omitempty"`
+	// PriorityClassName is used to set the PodSpec.PriorityClassName of the redpanda Statefulset.
+	PriorityClassName string `json:"priorityClassName,omitempty"`
```

Comment:
for the omitempty to have an effect string type needs to be changed to pointer to string

Reply:
It should not matter in this case, since if does not take effect, `{ "priorityClassName" :""}` is treated as priorityClassName not set.

Reply:
ok